### DISCUSSION TITLE
black MCDU & ND screens with the latest development version of A32nx

### DISCUSSION THREAD

**Discussion by alainzu** [flybywiresim/aircraft#7979] [Help]
Hello, I experience black MCDU & ND screens with the latest development version of A32nx.
I have read the reported known issues but no solution
I have tried a clean reinstall of development or stable version and using the fbw livery but no change.
I have cleaned my community folder of all liveries.
I have the latest version of MSFS 1.31.22.0
Screens come back if I use an old version of FBW A32nx od 2022???
Thanks for your help

**Comment by Pugwash984** [comment]
I have exactly the same problem but only with the MCDU on a remote display. Outline of the MCDU all there but screen blank - really do appreciate any ideas . the Flypad shows a disconnected wireless icon even though still using port 8380, all was working fine up until 3 months ago

**Reply by alainzu** [reply]
You have to clean your community folder of everything except FBW, PMDG, Fenix.... and you will solve the problem
Then you can try to reinstall freeware addons and see which of them interacts with A32nx cockpit view

**Reply by Pugwash984** [reply]
Thank you, I shall give it a try in a day or so, cheers

**Reply by Pugwash984** [reply]
Tried that - nothing works. I update my computer and MS Store every morning. There are only 4 files in the community folder which are ; flybywire aircraft, flybywire-externaltools-simbridge, and the 2 FSLTL files. Still the problem persists, Simbridge runs OK but the wireless icon in the flypad crossed and goes amber when pressed to connect. The MCDU will open in a new window but the screen is blank. I have now run out of ideas. My firewall settings have not changed sincer this started and everything is updated daily, and I mean every thing.

**Comment by mariokhan** [comment]
the problem is not solved?

> Попробовал - ничего не получается. Каждое утро я обновляю свой компьютер и MS Store. В папке сообщества есть только 4 файла, которые ; flybywire aircraft, flybywire-externaltools-simbridge и 2 файла FSLTL. Тем не менее, проблема сохраняется, Simbridge работает нормально, но значок беспроводной связи на флайпаде пересекается и становится желтым при нажатии для подключения. MCDU откроется в новом окне, но экран пуст. Сейчас у меня закончились идеи. Мои настройки брандмауэра не менялись с тех пор, как это началось, и все обновляется ежедневно, и я имею в виду все.
